netsim/augment/nodes.py

```diff
@@ -486,6 +486,19 @@ def add_host_static_routes(topology: Box) -> None:
     
     features = devices.get_device_features(n_data,topology.defaults)
     host_af_list = [ af for af in n_data.af if af != 'ipv6' or not features.initial.ipv6_use_ra ]
+    if 'dhcp' in topology.get('module',[]):       # If we use DHCP module, we have to scan for the DHCP clients
```

Comment:
Would it be possible to structure this such that the DHCP module performs this work? It breaks modularity

e.g. have the routing module create its static routes, but have the DHCP module remove them for clients

Reply:
See https://github.com/ipspace/netlab/pull/2043#issuecomment-2733652427